src/components/ListView.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
+import { db } from '../lib/firebase';
+import { useCollectionData } from 'react-firebase-hooks/firestore';
 
 const ListView = () => {
+  let token = localStorage.length !== 0 ? localStorage.getItem('token') : 'LA';
```

Comment:
Hey @ciradu2204 , we've updated our code so that the generated token is available across components. Before the merge, you would probably want to accept that token as a prop here instead of doing what you have on line 6.

Reply:
Thank you @katia-utochkina , I will work on it